internal/repository/index.go

```diff
@@ -26,27 +28,64 @@ type Index struct {
 }
 
 type indexEntry struct {
-	packID restic.ID
-	offset uint
-	length uint
+	packIndex int
```

Comment:
This really needs a comment: // Index into containing index's packs.

Reply:
I added a comment